src/com/dotmarketing/webdav/DotWebdavHelper.java

```diff
@@ -359,7 +371,10 @@ public Folder loadFolder(String url,User user) throws IOException{
 	}
 
 	public java.io.File loadTempFile(String url){
-		url = stripMapping(url);
+		try {
+			url = stripMapping(url);
+		} catch (IOException e) {
```

Comment:
I guess we don't want to stop the execution if the stripMapping fails but we should avoid empty catchs, maybe a log.info?